libs/components/packages/src/schematics/migrations/update-8/update-polyfill/update-polyfill.schematic.ts

```diff
@@ -13,21 +13,50 @@ function removePolyfillCode(
   project: ProjectDefinition
 ): Rule {
   return async (tree): Promise<Rule | void> => {
-    const targetsToUpdate: string[] = [];
+    // Saves a list of files that need to be checked for SKY UX polyfills.
+    const polyfillsFiles = new Set<string>();
+
+    // Saves a list of targets that use each polyfills file.
+    const targetsWithPolyfills = new Map<string, Set<string>>();
 
     // Check each target for 'polyfills' configuration.
     for (const [targetName, target] of project.targets.entries()) {
-      const polyfillsFile = target.options?.polyfills;
+      const files = [];
```

Comment:
👇 All of this new logic stems from an incorrect assertion in the unit test (read: stupid mistake on my part). See comment below.